Proteomics/tutorials/labelfree_vs_labelled.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: tutorial_hands_on
+topic_name: Label-free versus Labelled - How to Choose Your Quantitation Method
+tutorial_name: labelfree-vs-labelled
+---
+
+
+# Label-free versus Labelled - How to Choose Your Quantitation Method
+
+## Purpose
+A basic question when planning any quantitative proteomic experiment is the choice between label-free versus labelled quantitation. Here, we want to explain and discuss benefits and drawbacks of each method. 
+Finally, we will give a guideline what points may be vital for the decision in our opinion. We will not discuss different labelling methods here in detail, but focus on our basic question.
+
+A basic overview of different quantitation techniques is in the figure by Marc Vaudel given below:
+
+![Overview quantitation techniques](../images/Vaudel_label_vs_labelfree.png)
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. [Drawbacks and Benefits of Labelled Quantitation](#pro-contra)
+> 2. [Guideline: How to Choose Your Technique](#conclusion)
+
+<a name="pro-contra"/></a>
```

Comment:
Sorry, Björn just told me why it should be done this way. I will change it in a minute.